installer/manifests/keptn/templates/_helpers.tpl

```diff
@@ -47,7 +47,6 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 Selector labels
```

Comment:
general comments for this file: values of control plane and continuous delivery sub chart merged together and cleaned up